### ISSUE TITLE
Could not create folder under Windows

### ISSUE THREAD

**Issue by taoqf** [klaascuvelier/gulp-copy#36]
the issue comes with v4.0.0.
v1.0 is ok.

**Comment by klaascuvelier** [comment]
That's weird, nothing really changed in that area. (https://github.com/klaascuvelier/gulp-copy/blob/master/lib/gulp-copy.js#L81-L103)
I don't have access to a windows machine atm. What does the error say? Anything useful?

**Comment by taoqf** [comment]
```sh
E:\clone\by_jxc\node_modules\_gulp-copy@4.0.0@gulp-copy\lib\gulp-copy.js:66
                    throw new PluginError('gulp-copy', `Could not copy file <${file.path}>: ${error.message}`);
                    ^
Error: Could not copy file <E:\clone\by_jxc\src-app\apppp.json>: ENOENT: no such file or directory, open 'E:\clone\by_jxc\dist\apppp.json'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @feidao/daoke@1.0.2018 test-app: `gulp --gulpfile gulpfile-app.js weapp`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @feidao/daoke@1.0.2018 test-app script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Administrator\AppData\Roaming\npm-cache\_logs\2018-10-09T06_10_49_959Z-debug.log
```

**Comment by taoqf** [comment]
`\` maybe the problem.

**Comment by klaascuvelier** [comment]
Yeah, looks like my code uses `/` to join, but I'd guess the error should be different in that case.
But I guess I should make this work on windows as well.

Too bad circle-ci does not have windows support :(

**Comment by klaascuvelier** [comment]
Created this PR, have a look if you don't mind. https://github.com/klaascuvelier/gulp-copy/pull/37/files
Tests seem to work and the example works with the code as well.

**Comment by klaascuvelier** [comment]
@taoqf I published 4.0.1 with the slash fix. Can you check if that helps you out?

**Comment by klaascuvelier** [comment]
implemented by #37
